frontend/src/lib/components/ObjectTags/index.tsx

```diff
@@ -48,25 +48,29 @@ export function ObjectTags({
         }
     }, [saving])
 
+    console.log('TAGS', tags)
+
     return (
         <div style={style}>
-            {tags.map((tag, index) => {
-                return (
-                    <Tag key={index} color={COLOR_OVERRIDES[tag] || colorForString(tag)} style={{ marginTop: 8 }}>
-                        {tag}{' '}
-                        {!staticOnly &&
-                            onTagDelete &&
-                            (deletedTags.includes(tag) ? (
-                                <SyncOutlined spin />
-                            ) : (
-                                <CloseOutlined
-                                    style={{ cursor: 'pointer' }}
-                                    onClick={() => handleDelete(tag, tags, id)}
-                                />
-                            ))}
-                    </Tag>
-                )
-            })}
+            {tags
+                .filter((t) => !!t)
```

Comment:
Fixes an existing bug where we were trying to render undefined tags which led to white screening.